### DISCUSSION TITLE
Determine if string is empty?

### DISCUSSION THREAD

**Discussion by tiymat** [lmorg/murex#733] [Q&A] [answered]
I feel silly for asking such a basic question, I'm sure I'm missing something obvious. Is there a way to determine if a string is empty? (Either contains nothing, or contains nothing but whitespace characters.)

The best approach I've found is something like
```
tout bool false -> set strEmpty
try { out $strToTest -> regexp 'm/^$/'; set bool strEmpty = true }
```
but I'm sure there's an easier way.

**Comment by orefalo** [comment, marked as answer]
set vv = ""
$vv == ""  # return true

therefore

if { $vv == ""} then { out "it's empty" }  else { out "it's not" }

**Reply by tiymat** [reply]
Perfect, thanks!
